admin_guide/high_availability.adoc

```diff
@@ -677,10 +690,9 @@ that all the *router* VIPs point to the same *router*, and all the *geo-cache*
 VIPs point to the same *geo-cache* service. As long as one node remains running,
 all the VIP addresses are served.
 
-. Deploy the ipfailover router to monitor postgresql listening on node
+Deploy the ipfailover router to monitor postgresql listening on node
```

Comment:
1.I think should add a title for this step at least. or it is a strange when this step is come out suddentlly
2. the `--credentials` option is deprecated. please also update it.